blog-server-api/src/entities/tag.rs

```diff
@@ -1,8 +1,9 @@
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, Serialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Tag {
+    pub id: i64,
```

Comment:
u64 🤡

Reply:
changed, also for some methods and other entities